opm/material/fluidsystems/GenericOilGasFluidSystem.hpp

```diff
@@ -59,12 +59,12 @@ namespace Opm {
     class GenericOilGasFluidSystem : public BaseFluidSystem<Scalar, GenericOilGasFluidSystem<Scalar, NumComp> > {
     public:
         // TODO: I do not think these should be constant in fluidsystem, will try to make it non-constant later
-        static const int numPhases=2;
-        static const int numComponents = NumComp;
-        static const int numMisciblePhases=2;
+        static constexpr int numPhases=2;
```

Comment:
a bit inconsistent with the TODO comment. in any case, add spacing around =.

Reply:
Sure, will add the space. 

Regarding the TODO comments, it will be a totally separated refactoring/developing efforts when coming to reuse it for different number of phases.

Reply:
Maybe it will be part of the template parameter in some form.